### PR TITLE
[docs] update references to renamed zulu@17 homebrew cask

### PR DESCRIPTION
**Pull request by fobos531**
# Why

CleanShot 2024-05-03 at 18 14 58@2x

CleanShot 2024-05-03 at 18 15 19@2x

# How

Rename references to the updated cask

# Test Plan

# Checklist

- [X] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [X] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -177,4 +177,4 @@ To help keep CI green, please make sure of the following:
 
 ### Extra Credit
 
-- Our CI tests will finish early if you don't make changes to certain directories. If you want to **get results faster** then you should make changes to **docs** directory in one PR, and changes to anything else in another!
+- Our CI tests will finish early if you don't make changes to certain directories. If you want to **get results faster** then you should make changes to **docs** directory in one PR, and changes to anything else in another!
```
Comment (fobos531) [inline CONTRIBUTING.md:180]: not sure why this got marked as a change in git 🤔

Comment (raajnadar) [reply CONTRIBUTING.md:180]: Make by due to space to tab conversion or vice versa?

Comment (amandeepmittal) [reply CONTRIBUTING.md:180]: Maybe there's a Prettier formatting (although that seems unlikely here). No worries, on this, I'll check it out later.

**Review by amandeepmittal — APPROVED** [review]
Thanks for updating this @fobos531!

**Pull request merged**